geometry_msgs/Pose pose
geometry_msgs/Twist twist
sensor_msgs/LaserScan scan
float32[] target_distance
float32[] target_angle